chat/Makefile.am

```diff
@@ -0,0 +1,9 @@
+
+sbin_PROGRAMS = chat
+man8_MANS = chat.8
```

Comment:
```suggestion
dist_man8_MANS = chat.8
```
and remove the `EXTRA_DIST`

Reply:
Fixed, and revisited the other places this was used.